metals/src/main/scala/scala/meta/internal/metals/Compilations.scala

```diff
@@ -16,7 +16,7 @@ import ch.epfl.scala.bsp4j.BuildTargetIdentifier
 import ch.epfl.scala.{bsp4j => b}
 
 final class Compilations(
-    buildTargets: BuildTargets,
+    val buildTargets: BuildTargets,
```

Comment:
```suggestion
    buildTargets: BuildTargets,
```
this shouldn't be needed